content/departments/cloud/technical-docs/v1.1/mi1-1_delete_process.md

```diff
@@ -116,6 +116,8 @@ gcloud compute snapshots list --project=sourcegraph-managed-$CUSTOMER | grep "da
 ```sh
 # Under $CUSTOMER directory
 cd project
+
+terraform init # switch to local module
```

Comment:
Just to clarify: init is not even about switching to local module, it's about initialise `.terraform` folder locally, so pulling all providers, libraries etc and connecting to GCS state bucket to check if all is working ;)

Reply:
LOL, that makes more sense, I literally copied the other one which has this comment. Do you think it is more clear without any comment? (I do)